README.md

```diff
@@ -126,6 +127,13 @@ make
 sudo make install
 ```
 
+### Build options ###
```

Comment:
Will need to try and get the added dependencies for each of the various OS listed as per the initial build requirements.

Reply:
Yes, I will add name of the required packages to the README, thanks for reminding me.

Reply:
Done so now in a87d393aa90e0dda4e60d0b5d9daa7dd3c98254f, to the best of my knowledge and research